Homework 1:

dkrot-by opened the pull request:
- Сущности: - Подразделения - Работники (один работник относится к одному подразделению) - Платежные поручения (одно платежное поручение относится к одному работнику)
  - Хранение сущностей реализовано в отдельных списках
  - Реализовано API для управления сущностями:
    - POST: создание записи
    - GET: получение записи или списка всех записей - PUT: изменение записи - DELETE: удаление записи

dkrot-by committed:
Homework 1:
  - Сущности:
    - Подразделения
    - Работники (один работник относится к одному подразделению)
    - Платежные поручения (одно платежное поручение относится к одному работнику)
  - Хранение сущностей реализовано в отдельных списках
  - Реализовано API для управления сущностями:
    - POST: создание записи
    - GET: получение записи или списка всех записей
    - PUT: изменение записи
    - DELETE: удаление записи

makkov reviewed (approved):
Отличная работа 👍 
Единственное, в репозиториях я бы create заменил на save, а read на get или find.

dkrot-by committed:
Homework 1. Рефакторинг:
- В репозитерии изменены наименования методов: create -> save, read -> getAll, getBy...
- В сервисе и контроллере наименования методов: read -> getAll, getBy...
